pkgs/development/compilers/avian/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, zlib, jdk }:
+
+stdenv.mkDerivation {
+  name = "avian-1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "readytalk";
+    repo = "avian";
+    rev = "72c0547f4bdedc1ba8da2882317e3f758deab23f";
```

Comment:
Updated to use the version rather than the git sha.

Newbie question: why do I have to add `rec` after `stdenv.mkDerivation` to be able to use `${version}`?  Without adding `rec`, I get:

```
$ nix-env -f . -i avian
error: undefined variable ‘version’ at /home/james/code/nixpkgs/pkgs/development/compilers/avian/default.nix:4:19
```